app/queries/get_course_option_from_codes.rb

```diff
@@ -85,4 +90,18 @@ def self.build_course_option_error_message(record, match_count)
     error_message << " at site #{record.site.code}" if record.site
     error_message << " for course #{record.course.code}"
   end
+
+  def self.get_unique_site(record, attr, value)
+    sites = record.provider.sites.for_recruitment_cycle_years([record.recruitment_cycle_year]).where(code: value)
+
+    if sites.count > 1
+      record.errors.add(
+        attr,
+        "Found multiple sites with code: #{value} for provider: #{record.provider.code} in the current cycle.",
```

Comment:
We should use I18n for these messages, they are complex enough to be a PITA in tests etc.